### ISSUE TITLE
Get R1=90.00% and mAP=82.79%.

### ISSUE THREAD

**Issue by huo123lijuan** [jackie840129/STE-NVAN#4]
Hi,How can I get the best results?the code I should run? thank you.

**Comment by jackie840129** [comment]
Hi 
You can follow the **Usage-testing** part in Readme.

**Comment by huo123lijuan** [comment]
I have run the code:
1.train:
     CKPT=ckpt_NL_0230
     python3 train_NL.py --train_txt $TRAIN_TXT --train_info $TRAIN_INFO  --batch_size 64 \
                     --n_epochs 200 --lr 0.001 --lr_step_size 50 --optimizer adam --ckpt $CKPT --log_path   loss.txt --class_per_batch 8 \
                     --model_type 'resnet50_NL' --num_workers 8 --track_per_class 4 --S 8 --latent_dim 2048 --temporal Done  --track_id_loss \
                     --non_layers  0 2 3 0
2.evaluate:
LOAD_CKPT=.ckpt_NL_0230/ckpt_best.pth
python3 evaluate.py --test_txt $TEST_TXT  --test_info  $TEST_INFO   --query_info $QUERY_INFO \
                    --batch_size 64 --model_type 'resnet50_NL' --num_workers 8  --S 8 --latent_dim 2048 \
                    --temporal Done  --non_layers  0 2 3 0 --load_ckpt $LOAD_CKPT \
 
the result is very bad,CMC : 0.6374 , mAP : 0.4554,Is there anything wrong?

**Comment by hanzhs** [comment]
> I have run the code:
> 1.train:
> CKPT=ckpt_NL_0230
> python3 train_NL.py --train_txt $TRAIN_TXT --train_info $TRAIN_INFO --batch_size 64 
> --n_epochs 200 --lr 0.001 --lr_step_size 50 --optimizer adam --ckpt $CKPT --log_path loss.txt --class_per_batch 8 
> --model_type 'resnet50_NL' --num_workers 8 --track_per_class 4 --S 8 --latent_dim 2048 --temporal Done --track_id_loss 
> --non_layers 0 2 3 0
> 2.evaluate:
> LOAD_CKPT=.ckpt_NL_0230/ckpt_best.pth
> python3 evaluate.py --test_txt $TEST_TXT --test_info $TEST_INFO --query_info $QUERY_INFO 
> --batch_size 64 --model_type 'resnet50_NL' --num_workers 8 --S 8 --latent_dim 2048 
> --temporal Done --non_layers 0 2 3 0 --load_ckpt $LOAD_CKPT \
> 
> the result is very bad,CMC : 0.6374 , mAP : 0.4554,Is there anything wrong?

Same as mine.  Have you figure out where the problem is ?

**Comment by jackie840129** [comment]
Hi @hanzhs and @huo123lijuan
I found that I type the wrong learning rate in the script file.
I should be 1e-4 (0.0001)!

Thanks!
